command/git.go

```diff
@@ -26,6 +27,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+const defaultCloneDepth = 1000
```

Comment:
I did some testing of the mongo and mms repos and didn't see appreciable differences between time to clone with depth 1, 100, and 1000.